python-lib/tableau_server_utils.py

```diff
@@ -19,9 +19,13 @@ def get_project_from_name(server, project_name):
     :return: couple (Boolean{target project exists on server}, project)
     """
     page_nb = 1
-
     all_project_items, pag_it = server.projects.get(req_options=tsc.RequestOptions(pagenumber=page_nb))
-    pages_in_total = (pag_it.total_available // pag_it.page_size) + 1
+
+    partial_page = 1 if pag_it.total_available % pag_it.page_size > 0 else 0
+    pages_in_total = (pag_it.total_available // pag_it.page_size) + partial_page
+    logger.info(
+        f"Searching for project {project_name} in {pag_it.total_available} projects, querying {pages_in_total} pages ({partial_page} partial page) with page size: {pag_it.page_size}"
+    )
```

Comment:
Extract a method out of it

Reply:
Done in 3a514878784b2bac9c2bcb44d7e4c74a7ec1ea0d